### PR TITLE
Improve contributing docs

### PR DESCRIPTION
**Pull request by jd**
doc: add a note on pull-requests
doc: fix issue tracker link not being in RST

### CODE REVIEW

File: doc/source/contributing.rst [hunk 1]
```diff
@@ -14,6 +14,23 @@ Be sure to include a title and clear description, as much relevant information
 as possible, and a code sample or an executable test case demonstrating the
 expected behavior that is not occurring.
 
+.. _`GitHub issue tracker`: https://github.com/gnocchixyz/gnocchi/issues
```
Comment (chungg) [inline doc/source/contributing.rst]: should be part of previous commit

**Comment by jd** [comment]
Pull-request updated, HEAD is now c0b5b220a1fec7bd2c10345ed3ba833dc2547ccf

**Comment by chungg** [comment]
random question, but what are those "merge branch 'master'" commits? does that mean we'll have a bunch of extra commits for rebases?

**Comment by jd** [comment]
@chungg That's because in order to merge, the branch needs to be updated: either rebased by the author, or with master being merged in it via the UI. Once it's up to date and Travis has validated the PR, and the PR is Approved, then you can merge. :)

**Pull request merged**